app/services/IntestacyCoverSheetPdf.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
 const IntestacyPdf = require('./IntestacyPdf');
+const {trim, join} = require('lodash');
 
 class IntestacyCoverSheetPdf extends IntestacyPdf {
     post(formdata) {
```

Comment:
Can you use the FormatName.js for the name. thanks.

Reply:
Done for both intestacy and probate journeys.